apps/web/src/pages/ApplicantDashboardPage/components/DashboardHeading.tsx

```diff
@@ -0,0 +1,255 @@
+import * as React from "react";
+import { useIntl } from "react-intl";
+import {
+  BriefcaseIcon,
+  BookOpenIcon,
+  UsersIcon,
+  LightBulbIcon,
+  StarIcon,
+} from "@heroicons/react/24/solid";
+
+import { User } from "@gc-digital-talent/graphql";
+
+import Hero from "~/components/Hero/Hero";
+import useRoutes from "~/hooks/useRoutes";
+import { HeroCard } from "./HeroCard";
+import {
+  HeroCardExperienceItem,
+  HeroCardProfileItem,
+  ProfileItemStatus,
+} from "./HeroCardItem";
+import {
+  hasEmptyRequiredFields as aboutSectionHasEmptyRequiredFields,
+  hasEmptyOptionalFields as aboutSectionHasEmptyOptionalFields,
+} from "../../../components/UserProfile/ProfileSections/AboutSection";
+import {
+  hasEmptyRequiredFields as languageInformationSectionHasEmptyRequiredFields,
+  hasEmptyOptionalFields as languageInformationSectionHasEmptyOptionalFields,
+} from "../../../components/UserProfile/ProfileSections/LanguageInformationSection";
+import {
+  hasEmptyRequiredFields as governmentInformationSectionHasEmptyRequiredFields,
+  hasEmptyOptionalFields as governmentInformationSectionHasEmptyOptionalFields,
+} from "../../../components/UserProfile/ProfileSections/GovernmentInformationSection";
+import {
+  hasEmptyRequiredFields as workLocationSectionHasEmptyRequiredFields,
+  hasEmptyOptionalFields as workLocationSectionHasEmptyOptionalFields,
+} from "../../../components/UserProfile/ProfileSections/WorkLocationSection";
+import {
+  hasEmptyRequiredFields as workPreferencesSectionHasEmptyRequiredFields,
+  hasEmptyOptionalFields as workPreferencesSectionHasEmptyOptionalFields,
+} from "../../../components/UserProfile/ProfileSections/WorkPreferencesSection";
+import {
+  hasEmptyRequiredFields as diversityEquityInclusionSectionHasEmptyRequiredFields,
+  hasEmptyOptionalFields as diversityEquityInclusionSectionHasEmptyOptionalFields,
+} from "../../../components/UserProfile/ProfileSections/DiversityEquityInclusionSection";
+import {
+  hasEmptyRequiredFields as roleSalarySectionHasEmptyRequiredFields,
+  hasEmptyOptionalFields as roleSalarySectionHasEmptyOptionalFields,
+} from "../../../components/UserProfile/ProfileSections/RoleSalarySection";
```

Comment:
Not necessary but you should be able to do `import { ... } from "~/components/UserProfile/ProfileSectins/*.tsx";`

I got the auto import working for packages but it seems local imports are not using the alias 😢

Reply:
Yeah, it's tidier - [update imports](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5857/commits/f592bbed6469beb1e0a7bc2c94619e9b3b928b5d)